blogposts/2021/release-3.30.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Sourcegraph 3.30 release"
+publishDate: 2021-07-20T10:00-07:00
+description: "The Sourcegraph 3.30 release introduces"
+tags: [blog, release]
+slug: "release/3.30"
+published: false
+changelogItems:
+
+
+---
+
+Sourcegraph 3.30 is now available! For this release, we introduced
```

Comment:
Sentence is incomplete; was that intentional? Maybe we just need a ":" at the end?
```suggestion
Sourcegraph 3.30 is now available! For this release, we introduced:
```